setup.py

```diff
@@ -30,6 +30,7 @@
             "GitPython==3.1.7",
             "bandit",
             "jsonschema",
+            "parameterized"
```

Comment:
no need for that, I think we actually can remove the whole dev block, we don't use it anywhere and there are so many missing libs here 🙈